pyalarmdotcomajax/pyalarmdotcomajax.py

```diff
@@ -180,6 +191,13 @@ async def _async_get_system_info(self):
             self._partitionid = json["data"]["relationships"]["partitions"]["data"][0][
                 "id"
             ]
+
+            locks = json["data"]["relationships"].get("locks", {}).get("data", [])
+
+            self._lock_detected = len(locks) > 0
+            if self._lock_detected:
+                self._lockid = json["data"]["relationships"]["locks"]["data"][0]["id"]

```

Comment:
@vegardengen @uvjustin I noticed this code assumes you have only 1 lock on your system. In my case I have two - are either of you planning on adding support for multiple locks? I can take a stab at support if not.

Reply:
Thanks for the message. I don't have any locks myself so would be great if you or @vegardengen could do this.

Reply:
@kevin-david unfortunately, I have only one lock, so it would be difficult for me to test multi-lock support. I think it would be great if you did it.

Reply:
Done - please take a look!

- https://github.com/uvjustin/pyalarmdotcomajax/pull/17 adds the required support in the library
- https://github.com/uvjustin/alarmdotcom/pull/63 consumes the library changes and adds each lock as an entity